features/makepot.feature

```diff
@@ -3682,6 +3682,51 @@ Feature: Generate a POT file of a WordPress project
       msgid "My style variation"
       """
 
+  Scenario: Extract strings from the styles.blocks.variations section of theme.json files
+    Given an empty foo-theme directory
+    And a foo-theme/theme.json file:
+      """
+      {
+        "version": "1",
+        "settings": {
+          "color": {
+            "duotone": [
+                { "slug": "dark-grayscale", "name": "Dark grayscale", "colors": [] }
+            ],
```

Comment:
This comma is invalid JSON. Removing it at https://github.com/wp-cli/i18n-command/pull/405/commits/c918b7ab629439fb5caf731985bc0fbadafde361